onnxruntime/test/util/test_utils.cc

```diff
@@ -69,6 +69,17 @@ int CountAssignedNodes(const Graph& current_graph, const std::string& ep_type) {
 void RunAndVerifyOutputsWithEP(const ORTCHAR_T* model_path, const char* log_id,
                                std::unique_ptr<IExecutionProvider> execution_provider,
                                const NameMLValMap& feeds) {
+  // read raw data from model provided by the model_path
+  ONNX_NAMESPACE::ModelProto model_proto;
+  ASSERT_STATUS_OK(onnxruntime::Model::Load(model_path, model_proto));
+  std::string model_data;
+  model_proto.SerializeToString(&model_data);
+  RunAndVerifyOutputsWithEPViaModelData(model_data, log_id, std::move(execution_provider), feeds);
+}
+
+void RunAndVerifyOutputsWithEPViaModelData(const std::string& model_data, const char* log_id,
```

Comment:
This will be better to take a gsl::span<uint8_t> instead of std::string

Reply:
thanks for the suggestion! modified it.